pkg/config/config-map_test.go

```diff
@@ -25,13 +25,12 @@ import (
 
 	"kubevirt.io/kubevirt/pkg/libvmi"
 
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
 	k8sv1 "k8s.io/api/core/v1"
 
-	"kubevirt.io/client-go/api"
-
 	v1 "kubevirt.io/api/core/v1"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
```

Comment:
Can I ask you to follow the [coding convention](https://github.com/kubevirt/kubevirt/blob/main/docs/coding-conventions.md#uniform-import-order-and-naming) for the imports? :)

Reply:
fixed them